open_discussions_api/channels/client_test.py

```diff
@@ -0,0 +1,76 @@
+"""Tests for user client API"""
+import json
+
+import pytest
+
+
+def test_create_channel(api_client):
+    """test a happpy path"""
+    resp = api_client.channels.create(
+        title="new test channel",
+        name="examplechannel",
+        public_description="a good channel for all things",
+        channel_type="public"
+    )
+    assert resp.status_code == 201
+    assert json.loads(resp.request.body) == {
+        "title": "new test channel",
+        "name": "examplechannel",
+        "public_description": "a good channel for all things",
+        "channel_type": "public"
+    }
+    assert resp.json() == {
+        "title": "new test channel",
+        "name": "examplechannel",
+        "public_description": "a good channel for all things",
+        "channel_type": "public"
+    }
```

Comment:
why do we need both line 16 and 22?

Reply:
the former basically asserts that our request to the external service (open-discussions) is well-formed, and the latter basically is just asserting that the response from open-discussions correlates with that as we expect. Nathan has similar tests in the user module, which is why I went ahead and added them.

Reply:
sorry, I did not realize that the first was the request

Reply:
:)